_data/harnesses/console-consumer-producer-avro/kafka.yml

```diff
@@ -33,8 +33,14 @@ dev:
 
     - title: Create a schema
       content:
-          - action: execute
-            file:
+          - change_directory: console-consumer-producer-avro
+            action: execute
+            file: tutorial-steps/dev/make-schema-dir.sh
```

Comment:
added a step to explicitly create the schema directory. Technically not required, but it fits the model we do with our other KT's